tests/functional/context_methods/test_log_arbitrary_types.py

```diff
@@ -0,0 +1,55 @@
+import pytest
```

Comment:
Not sure if this is even the appropriate variation of test for this part of the codebase, I found this directory through randomly wandering, and then I copied adjacent files to get it to work.